templates/homepage.html

```diff
@@ -132,9 +135,12 @@
                         <i class="thumbs up icon status-icon"></i>
                     </div>
                     <div class="twelve wide column">
-                       <p class="status-head">1007pp</p> <!-- $d.pp-->
+                        {{ $vanilla_pp := (rediget "akatsuki:top:scores:pp") }}
+                        {{ $vanilla_id := (rediget "akatsuki:top:scores:id") }}
+                        {{ $vanilla_name := (rediget "akatsuki:top:scores:name") }}
+                       <p class="status-head">{{ $vanilla_pp }}pp</p>
                        <p class="status-footer">Top score (Vanilla)</p>
-                       <p class="status-head" style="font-size: 12px!important;">Played by <a href="/u/1176">RyuK</a></p>
+                       <p class="status-head" style="font-size: 12px!important;">Played by <a href="/u/{{ $vanilla_id }}?rx=1">{{ $vanilla_name }}</a></p>
```

Comment:
i think, being that this is vanilla, we'd want to send onclick to vanilla profile
```suggestion
                       <p class="status-head" style="font-size: 12px!important;">Played by <a href="/u/{{ $vanilla_id }}?rx=0">{{ $vanilla_name }}</a></p>
```

Reply:
oh yep thats a copy paste i messed up lol